oneview/resource_logical_enclosure.go

```diff
@@ -82,15 +83,18 @@ func resourceLogicalEnclosure() *schema.Resource {
 
 			"enclosure_group_uri": {
 				Type:     schema.TypeString,
-				Required: true,
+				Computed: true,
+				Optional: true,
 			},
 			"enclosure_uris": {
-				Required: true,
+				Computed: true,
+				Optional: true,
 				Type:     schema.TypeList,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
 			"firmware": {
 				Optional: true,
+				Computed: true,
```

Comment:
why have we set firmware to computed when all the attributes inside it are optional.

Reply:
done